# Genomic-DNA-Extractions-for-Kelletia-Kelletii


Benjamin N. Daniels

California Polytechnic State University
<br>

## Abstract
Commands and scripts used for Genomic DNA Extraction Optimization and Validation for Marine Invertebrate Tissue
<br>
<br>

<img width="1194" alt="Screenshot 2023-07-21 at 3 16 26 PM" src="https://github.com/bndaniel/Genomic-DNA-Extractions-for-Kelletia-Kelletii/assets/71898958/8ac48793-e3dd-42a3-8358-595f1569a21b">
<br>
Figure: Assessment of 252 randomly selected DNA extractions  A) Examples of Digested, HMW + Degradation, and HMW quality genomic DNA extractions determined by gel electrophoresis B) Quality assessment of genomic DNA extractions. Average concentration of Digested extractions (22 total samples), HMW + Degradation extractions (75 total samples), and HMW extractions (155 total samples). The top of the box represents the 75th percentile and the bottom represents the 25th percentile. Whiskers represent the highest and lowest values of the sample group. Kruskal Wallis tests are indicated by horizontal lines and asterisks (****: p≤0.0001, **: p≤0.01) (Table S2). C) Pie chart displaying percentage of the 252 randomly-selected samples of the 3,325 DNA extractions within each quality assessment group: Digested, HMW + Degradation or HMW.
  <br>

<img width="1367" alt="Screenshot 2023-07-21 at 3 16 43 PM" src="https://github.com/bndaniel/Genomic-DNA-Extractions-for-Kelletia-Kelletii/assets/71898958/c7bddd52-ce86-48d0-93ba-e35e92871cf4">
  <br>
Figure: A) GT-seq tested samples in order of percent genotyped (%GT) from scaled DNA extractions. 96 total samples were tested (see sup. data). Varied read distribution indicates unsuccessful sequencing especially in samples 40+. B)  GT-seq on the same 96 samples after Zymo clean and concentrator. Samples are ordered by %GT. Distribution of raw reads across samples indicates much more consistent and successful sequencing results. C) Samples tested on RAD-seq platform. Reads that passed the quality filter and contained the RADcutsite are indicated by Retained reads. Reads are ordered by retained reads. Overall, an average of 78.84% of reads were retained after filtering for quality and RADcutsite. 94.77% of samples produced > 180,000 raw reads.
  <br>
